jenkins/bb-fuel.groovy

```diff
@@ -52,6 +53,7 @@ pipeline {
                                     "-Duse.pfm.categories.for.transactions=${params.USE_PFM_CATEGORIES_FOR_TRANSACTIONS} " +
                                     "-Dingest.approvals.for.payments=${params.INGEST_APPROVALS_FOR_PAYMENTS} " +
                                     "-Dingest.approvals.for.contacts=${params.INGEST_APPROVALS_FOR_CONTACTS} " +
+                                    "-Dingest.approvals.for.contacts=${params.INGEST_APPROVALS_FOR_NOTIFICATIONS} " +
```

Comment:
ingest.approvals.for.**notifications**=

Reply:
yes, that was my copy / paste issue, thank you!